retrospective-update/download_era5.py

```
@@ -0,0 +1,139 @@
+import traceback
+from datetime import datetime
+
+import cdsapi
+import numpy as np
+import pandas as pd
+import xarray as xr
+from dateutil import relativedelta
+from natsort import natsorted
+
+from cloud_logger import CloudLog
+from set_env_variables import *
```

Ruff (0.12.2)

`from set_env_variables import *` used; unable to detect undefined names

(F403)